content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md

```diff
@@ -73,7 +72,77 @@ telemetry agents on the node, make sure to check with the vendor of the agent wh
 
 ### Telemetry and security agent vendors
 
+{{% thirdparty-content %}}
+
+This section is intended to collect information about various telemetry and security agents that may have dependency on containers runtime. Support matrix section outlines the supported runtimes and Migration from dockershim section is designed to help users transition from dockershim to other container runtimes.
```

Comment:
The second statement contains some pointers to nowhere. Please fix.

Reply:
@AugustasV ptal at @tengqm 's comment